applications/IgaApplication/python_scripts/assign_vector_variable_and_constraints_to_conditions_process.py

```diff
@@ -57,9 +57,9 @@ def __init__(self, model, settings ):
                 component_params.AddEmptyValue("variable_name").SetString(variable_name + variable_dir)
                 component_params.AddValue("local_axes",settings["local_axes"])
                 self.aux_processes.append( AssignScalarVariableToConditionsProcess(model, component_params) )
-                KratosMultiphysics.VariableUtils().SetFlag(flag_name, True, self.model_part.Conditions)
+                KratosMultiphysics.VariableUtils().SetFlag(eval(flag_name), True, self.model_part.Conditions)
```

Comment:
`eval` should be avoided if possible

can you explain this change?

Reply:
I plan to neglect ```the_flag``` and passing ```flag_name``` directly in lines 60 & 62. 

Since ```eval``` should be avoided, we'll figure another way.

Reply:
my question is more if `KratosMultiphysics.KratosGlobals.GetFlag(flag_name)` doesn't work any more?

Reply:
Yeah, ```KratosMultiphysics.KratosGlobals.GetFlag(flag_name)``` doesn't work.

Reply:
What is the error?
It doesn't find the flag?

Reply:
Yes, it doesn't find the flag  (```"KratosMultiphysics.IgaApplication.IgaFlags.FIX_" + variable_component_name```), and it ends with segmentation fault. I am still figuring about it as well.

Reply:
hm ok
Then I guess I can live with the `eval`

Reply:
All right, thanks!